test_import_filtering.py

```
@@ -0,0 +1,63 @@
+import fake_module
+from fake_module import something
+"""
+
+another_string = """
+This is also a triple-quoted string
+import another_fake_module
+from yet_another_fake import stuff
+"""
+
+
+def example_function():
```

Ruff (0.11.9)

Missing return type annotation for public function `example_function`

(ANN201)